yarn.lock

```diff
@@ -9207,6 +9207,13 @@ ovh-api-services@^6.26.0:
   dependencies:
     lodash "^3.10.1"
 
+ovh-api-services@^6.24.1:
+  version "6.24.1"
```

Comment:
Sounds a bit strange because you explicitly say this previously:

```diff
diff --git a/packages/manager/apps/public-cloud/package.json b/packages/manager/apps/public-cloud/package.json
index 3c1b04b42..2efb1e4e6 100644
--- a/packages/manager/apps/public-cloud/package.json
+++ b/packages/manager/apps/public-cloud/package.json
@@ -55,7 +55,7 @@
     "oclazyload": "^1.1.0",
     "ovh-angular-pagination-front": "ovh-ux/ovh-angular-pagination-front#^5.1.0",
     "ovh-angular-q-allsettled": "ovh-ux/ovh-angular-q-allSettled#^0.3.1",
-    "ovh-api-services": "^6.26.0",
+    "ovh-api-services": "^6.28.0",
     "ovh-common-style": "^5.0.0",
     "ovh-manager-webfont": "^1.2.0",
     "ovh-ui-angular": "^3.2.2",
diff --git a/packages/manager/modules/pci/package.json b/packages/manager/modules/pci/package.json
index 46a673e65..111f96351 100644
--- a/packages/manager/modules/pci/package.json
+++ b/packages/manager/modules/pci/package.json
@@ -61,7 +61,7 @@
     "oclazyload": "^1.1.0",
     "ovh-angular-pagination-front": "ovh-ux/ovh-angular-pagination-front#^5.1.0",
     "ovh-angular-q-allsettled": "ovh-ux/ovh-angular-q-allSettled#^0.3.1",
-    "ovh-api-services": "^6.26.0",
+    "ovh-api-services": "^6.28.0",
     "ovh-common-style": "^5.0.0",
     "ovh-manager-webfont": "^1.1.0",
     "ovh-ui-angular": "^3.2.2",
```